src/port.js

```diff
@@ -0,0 +1,19 @@
+class Port {
```

Comment:
The formatting of this file is a little bit off. 
you should leave a clear line between each method, and delete the empty lines, line 14 and 18.
Indentation of the _addShip_ and _removeShip_ should be alined with the constructor.

````
class Port {
  constructor(Port) {
    this.name = Port;
    this.ships = [];
  }

  addShip(Ship) {
    this.ships.push(Ship);   
  }

  removeShip(Ship) {
    const index = this.ships.indexOf(Ship);
    this.ships.splice(index, 1);
  }
}
`````